packages/hash/external-services/docker-compose.yml

```diff
@@ -43,14 +42,50 @@ services:
     environment:
       - PGDATA=/data/pgdata
     ports:
-      - 5432:5432
+      - "${POSTGRES_PORT}:5432"
     volumes:
       - ../../../var/external-services/postgres/data:/data
-
+      - ./postgres/postgresql.conf:/etc/postgresql/postgresql.conf:ro
+      - ./postgres/init-user-db.sh:/docker-entrypoint-initdb.d/init-user-db.sh
   hash-dev-redis:
     deploy:
       restart_policy:
         condition: on-failure
     image: redis:6.2
     ports:
       - 6379:6379
+  kratos-migrate:
+    depends_on:
+      - postgres
+    image: oryd/kratos:v0.10.1
+    volumes:
+      - type: bind
+        source: ./kratos
+        target: /etc/config/kratos
+    environment:
+      - DSN=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@postgres:${POSTGRES_PORT}/${HASH_KRATOS_PG_DATABASE}
+    command: -c /etc/config/kratos/kratos.yml migrate sql -e --yes
+  kratos:
+    depends_on:
+      - kratos-migrate
+    image: oryd/kratos:v0.10.1
+    ports:
+      - "4433:4433" # public
+      - "4434:4434" # admin
+    restart: unless-stopped
+    environment:
+      - SECRETS_COOKIE=PLEASE-CHANGE-ME-I-AM-VERY-INSECURE
```

Comment:
When are we going to do these 😁 Is there a todo or Asana task?

Reply:
My understanding is that this `docker-compose.yml` file is for dev purposes only, so don't think it should be changed here. If my understanding is correct, should I change this value here to avoid future confusion?

Reply:
I'm not sure how we intend to use this, but if it is dev only I'd maybe change it to something equally as silly (but less misleading) like `VERY-INSECURE-AND-SHOULD-ONLY-BE-USED-IN-DEV` or something

Reply:
Sure, have addressed in https://github.com/hashintel/hash/pull/889/commits/461129c74284495245d61622f54bf09ff478e1cd